MAUI/DataGrid/Column-sizing.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: Column Sizing in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Column Sizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more.
+platform: maui
+control: SfDataGrid
+documentation: ug
+---
+
+# Column Sizing in .NET MAUI DataGrid (SfDataGrid)
+
+DataGrid allows you to set the column widths based on certain logic using `SfDataGrid.ColumnWidthMode` or `DataGridColumn.ColumnWidthMode`property. Below is the list of predefined column sizing options available.
+
+<table>
+<tr>
+<th>
+Type
+</th>
+<th>
+Column width
+</th>
+</tr>
+<tr>
+<td>
+<code>Fill</code>
+</td>
+<td>
+Divides the total width equally for columns. 
+</td>
+</tr>
+<tr>
+<td>
+<code>Auto</code>
+</td>
+<td>
+Calculates the width of column based on header and cell contents. So that header and cell content’s are not truncated.  
```

Comment:
Calculates the width of columns based on header and cell contents so that header and cell contents are not truncated.